frontend/src/components/workspace/users-settings/user-item/user-item.tsx

```diff
@@ -0,0 +1,23 @@
+import { getAllowedClasses } from '../../../../helpers/dom/dom';
+import styles from '../styles.module.scss';
+import { IWorkspaceUser } from '../../../../common/interfaces/workspace';
```

Comment:
Update all nested imports 
```suggestion
import { IWorkspaceUser } from 'common/interfaces/workspace';
```